tidy3d/plugins/invdes/region.py

```diff
@@ -160,30 +172,6 @@ def params_shape(self) -> typing.Tuple[int, int, int]:
         num_pixels[np.isinf(num_pixels)] = 1
         return tuple(int(n) for n in num_pixels)
 
-    def params_uniform(self, value: float) -> np.ndarray:
```

Comment:
is it worth just keeping these but throwing deprecation warnings? guess for this demo it's not necessary..

Reply:
do we replace them with dummy methods? because we can't really sample anything other than the parameter spec that was provided. or do keep the full methods around?

Reply:
ok keeping the methods around and warning users for now that we'll remove it